src/Cake.Core/CakeEngine.cs

```diff
@@ -59,6 +59,7 @@ public CakeTaskBuilder<ActionTask> RegisterTask(string name)
                 const string format = "Another task with the name '{0}' has already been added.";
                 throw new CakeException(string.Format(CultureInfo.InvariantCulture, format, name));
             }
+
```

Comment:
This is probably also a StyleCop issue not related to this PR?

Reply:
No, I just tidied up a little :D

Reply:
Will skip this change.